compiler/src/dotty/tools/dotc/core/StdNames.scala

```diff
@@ -944,6 +949,8 @@ object StdNames {
     final val JavaSerializable: N    = "java.io.Serializable"
   }
 
+
+
   class JavaTermNames extends JavaNames[TermName] {
```

Comment:
1000

Reply:
Is this when it became necessary to report bots on github?